src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
+    private static final int NUMBER_FOR_RANDOM = 10;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[randomInt()];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = FIGURE_SUPPLIER.getRandomFigure();
+        }
+        for (int i = 0; i < figures.length; i++) {
+            System.out.println(figures[i].toString());
+        }
+    }
+
+    private static int randomInt() {
```

Comment:
It's bad practice to use `static` methods, let's make it non static. Have you read the article I threw you about the names of the methods (this one: https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names)? What is the correct method name?

Reply:
I changed toString() - drawFigure().

I crated a new class 'RandomNumberFigures', because we can't to use non-static in static method (main is static).